radicle-node/src/wire/protocol.rs

```diff
@@ -568,21 +568,37 @@ where
                 // TODO: This should be a fatal error, there's nothing we can do here.
                 log::error!(target: "wire", "Can't poll connections: {}", err);
             }
-            reactor::Error::ListenerPollError(id, err) => {
+            reactor::Error::ListenerPollError(id, _) => {
                 // TODO: This should be a fatal error, there's nothing we can do here.
-                log::error!(target: "wire", "Received error: listener {} disconnected: {}", id, err);
+                log::error!(target: "wire", "Received error: listener {} disconnected", id);
```

Comment:
Were the errors too much noise or not useful?

Reply:
They actually weren't errors, was a bad variable name, they are the event state of the file descriptor, which is kind of useless.